Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM debian:stretch
+MAINTAINER James McFeeters "jmcfeeters@usgs.gov"
+RUN apt-get update && apt-get install -y \
+    python3-pip \
+    nginx-light \
+    curl \
+    apt-transport-https \
+    git
+
+COPY . /app
+WORKDIR /app
+
+RUN cp DOIRootCA2.cer /etc/ssl/certs/DOIRootCA2.pem \
+&& cat DOIRootCA2.cer >> /etc/ssl/certs/ca-certificates.crt
+
+RUN rm -f /etc/apt/sources.list.d/chris-lea-node_js-jessie.list \
+&& curl --cacert DOIRootCA2.cer https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
+&& echo 'deb https://deb.nodesource.com/node_6.x jessie main' > /etc/apt/sources.list.d/nodesource.list \
+&& apt-get update \
+&& apt-get install -y nodejs
+
+RUN pip3 install -r requirements.txt
```

Comment:
I thought that this was working because I was storing the DOI cert with the system certs, but it still works with those lines commented out and without pointing it at the DOI certificate. I think I'll add a `--cert` flag just to be safe.